test/dune

```diff
@@ -0,0 +1,37 @@
+(* -*- tuareg -*- *)
+
+let flags_gen = function
+  | [] -> ""
+  | pkgs ->
+    let cmd = "ocamlfind ocamlc -verbose" ^ (
```

Comment:
I think this is not needed anymore, see here: https://github.com/jonludlam/xcp-idl/blob/a6ae5782b7ab9c950eff52b01d186541619e0c8b/xen/jbuild
All this can be done much more simply with `(preprocess (pps ...))` now

Reply:
Hmmm, I have already removed the unix requirement, in fact I've replaced these with Jbuild_plugin calls as per the dune docs.